### PR TITLE
#164711270 ; viewer channel names

### PR DESCRIPTION
**Pull request by lawhead**
Also includes adjustments for a more consistent spacing between the names and yaxis ticks.

### CODE REVIEW

File: bcipy/gui/viewer/data_viewer.py [hunk 1]
```diff
@@ -154,16 +156,22 @@ def init_axes(self):
         for i, channel in enumerate(self.data_indices):
             ch_name = self.channels[channel]
             axes[i].set_frame_on(False)
-            axes[i].set_ylabel(ch_name, rotation=0, labelpad=15)
+            axes[i].set_ylabel(
+                ch_name,
+                rotation=0,
+                labelpad=self.yaxis_label_space,
+                fontsize=14)
```
Comment (tab-cmd) [inline bcipy/gui/viewer/data_viewer.py]: I'd pull up font size to the instance variables

File: bcipy/gui/viewer/data_viewer.py [hunk 2]
```diff
@@ -154,16 +156,22 @@ def init_axes(self):
...
                      for sec in tick_names]
             axes[i].xaxis.set_major_locator(ticker.FixedLocator(ticks))
             axes[i].xaxis.set_major_formatter(
                 ticker.FixedFormatter(tick_names))
-            # axes[i].yaxis.set_major_locator(NullLocator())
-            # axes[i].yaxis.set_major_formatter(NullFormatter())
+            axes[i].tick_params(axis='y', which='major', labelsize=10)
+            for tick in axes[i].get_yticklabels():
+                # fixed width font so we can adjust spacing predictably
+                tick.set_fontname('DejaVu Sans Mono')
```
Comment (tab-cmd) [inline bcipy/gui/viewer/data_viewer.py]: I'd also pull the fontname up

**Review by tab-cmd — APPROVED** [review]
Looks good! Please pull up those variables I commented on when you get the chance

File: bcipy/gui/viewer/data_viewer.py (hunk 2 above)
Comment (lawhead) [reply bcipy/gui/viewer/data_viewer.py]: Thanks, good suggestions.

**Pull request merged**